src/components/app/app.js

```diff
@@ -35,23 +41,44 @@ export class AppComponent {
       onAboutClick: this.handleAboutOnclick.bind(this)
     });
     this.toolBarComponent.mount();
-    this.drawerComponent = new DrawerComponent(this.drawerMountPoint, {
-      onAddAccountClick: this.handleAddAccountClick.bind(this),
-      onAddTagClick: this.handleAddTagOnclick.bind(this)
-    });
-    this.drawerComponent.mount();
+    this.initDrawer();
     this.transactionsComponent = new TransactionsComponent(
       this.transactionsMountPoint
     );
     this.transactionsComponent.mount();
     this.addAccountDialog = new AddAccountComponent(this.addAccountMountPoint);
     this.addAccountDialog.mount();
-    this.addTagDialog = new AddTagComponent(this.addTagMountPoint);
-    this.addTagDialog.mount();
     this.aboutDialog = new AboutComponent(this.aboutMountPoint);
     this.aboutDialog.mount();
   }
 
+  mountTransactionsComponent(tags) {
+    this.transactionsComponent = new TransactionsComponent(
+      this.transactionsMountPoint,
+      { tags: tags }
```

Comment:
what does this method do? You don't need to do anything with transaction component. your sphere of work is Drawer. TransactionsComponent is a content part. 
And by using this method you rewrite `this.transactionsComponent` from row 45